docs/antora/modules/ROOT/pages/setup/source.adoc

```diff
@@ -0,0 +1,342 @@
+
+If you are unsure where to find your `my.cnf`, use the following command:
+
+ $ mysql --verbose --help | grep my.cnf
+
+Add the following at the end of the file:
+
+ default_authentication_plugin=mysql_native_password
+
+Save and exit.
+
+Next, login via the terminal:
+
+ $ mysql -uroot
+
+Then run the following command to update the root password:
+
+ ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '';
+
+Now you should be able to login to your MySQL 8 via your MySQL Client.
+
+
+--
+====
+
+=== Redis
+
+{url_redis}[Redis] is used for caching temporary data.
+
+[tabs]
+====
+Redis with Docker::
++
+--
+**Docker Setup**
+
+Follow the steps described in the xref:{url_setup_docker_docker}[Prerequisites > Docker] section of the "Installation with Docker" page.
+
+**Installation**
+
+.How to install and start Redis with Docker
+[source,bash]
+----
+docker run --name redis_service --port 6379:6379 -d redis:5-alpine
+----
+.How to use the custom redis.conf file
+[source,bash]
+----
+docker run --name redis_service -v /path/to/custom_redis.conf:/usr/local/etc/redis/redis.conf --port 6379:6379 -d redis:5-alpine
+----
+The above commands should be enough to start Redis which is ready to use with Lisk Service.
+
+To stop the Docker container again, execute the following commands below:
+
+.How to stop Redis with Docker
+[source,bash]
+----
+docker stop redis_service
+----
+--
+Redis system-wide::
++
+--
+*Ubuntu*
+
+[source, bash]
+----
+sudo apt-get install redis-server
+----
+
+*macOS*
+
+[source, bash]
+----
+brew install redis
+----
+--
+====
+
+
+////
+Lisk Service is not compatible with this service right now.
+we should encourage community to make Lisk Service compatible with this service, then they can use it as alternative GeoIP service.
+=== GeoJS
+
+{url_geojs}[GeoJS] is used by the Network Monitor for IP address geo-location.
+
+[source,bash]
+----
+#todo
+----
+////
+
+
+=== PM2
+
+{url_pm2}[PM2] helps manage the node processes for Lisk Service and also offers easy log rotation (Highly Recommended).
+
+[source,bash]
+----
+npm install -g pm2
+----
+
+== Installation
+
+[IMPORTANT]
+====
+It is strongly recommended that you synchronize your Lisk Core node with the network **before** starting the Lisk Service.
+====
+
+If you have not already done so, clone the {url_github_service}[lisk-service^] GitHub repository and then navigate into the project folder and check out the latest release.
+
+[source,bash]
+----
+# Clone Lisk Service repository
+git clone https://github.com/LiskHQ/lisk-service.git
+
+# Change directory to the new repository
+cd lisk-service
+
+# Switch to the recent stable as a base
+git checkout v0.7.0
+
+# ...or use the development branch
+git checkout development
+----
+
+Install all npm dependencies from the root directory.
+
+[source,bash]
+----
+make build-local
+----
+
+Now it is possible to start Lisk Service.
+
+.Start Lisk Service from Source code
+[source,bash]
+----
+npm run start
```

Comment:
```suggestion
.Start Lisk Service from source code using pm2 from the root directory.
[source,bash]
----
pm2 start ecosystem.config.js
```

Reply:
I think `npm run start` will do the same. Ref: https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/package.json#L27

Reply:
Then, we can leave it as is. @chris529 please ignore the suggestion.